### PR TITLE
may dev update

### PR DESCRIPTION
**Pull request by korydraughn**

### CODE REVIEW

File: content/posts/irods-development-update-may-2022.md [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+
+
+Hello again! Time for another monthly development update.
+
+iRODS 4.3.0 is very close. Delay server migration is now possible. With that comes new `iadmin` subcommands for setting the delay server and determining who is running the delay server. We've successfully ported the legacy Kerberos authentication plugin to the new authentication plugin framework. Issues with compilation of Python byte-code and RPM package generation have been resolved as well. Who would've guessed that was a thing.
+
+The Development Environment is receiving some enhancements in the form of debuggers and runner images for Ubuntu 20, Debian 11, and Almalinux 8. And since we're talking about Docker-based tools, you'll be happy to know that support for SSL is being added to the Testing Environment's federation test infrastructure.
+
+Finishing touches are being applied to the Indexing plugin. Once released, it will support `iput --metadata` and user/group ownership metadata. Not only that but, configuration settings that take integers as strings will also be accepted as actual integers. We know passing integers around as strings is a little weird. We hope this change improves the experience a little.
```
Comment (trel) [inline content/posts/irods-development-update-may-2022.md:14]: "Not only that but, configuration settings that take integers as strings will also be accepted as actual integers. We know passing integers around as strings is a little weird. We hope this change improves the experience a little."

is a bit confusing/awkward - consider...

"Not only that, but the handling of configuration settings is now a bit more flexible regarding integers and strings."

File: content/posts/irods-development-update-may-2022.md [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+
+Something that we haven't spoke about much is the R client library. Given the challenges around creating a standalone compilable R package, the idea of using the iRODS C++ REST API in R has been floated as an alternative. This would reduce the dependencies for the R client library and make it much easier to expose it to the R community. If you have thoughts/concerns/ideas about that process, please reach out to us via email or open an issue at https://github.com/irods/irods_client_library_r_cpp.
```
Comment (trel) [inline content/posts/irods-development-update-may-2022.md:16]: spoke -> spoken

File: content/posts/irods-development-update-may-2022.md [hunk 3]
```diff
@@ -0,0 +1,286 @@
...
+    - MinIO has deprecated the gateway interface
+    - discussion has shifted focus to custom S3 implementation
+    - working meeting
+        - Fri, August 5, 2022, 3pm ET
+
+- [TRiRODS](https://irods.org/trirods)
+
+    - Wed, May 18, 2022, 4pm ET
+    - speaker, TBD
+
```
Comment (trel) [inline content/posts/irods-development-update-may-2022.md:58]: can remove this for this posting

Comment (korydraughn) [reply content/posts/irods-development-update-may-2022.md]: done

File: content/posts/irods-development-update-may-2022.md (hunk 2 above)
Comment (korydraughn) [reply content/posts/irods-development-update-may-2022.md]: done

File: content/posts/irods-development-update-may-2022.md (hunk 1 above)
Comment (korydraughn) [reply content/posts/irods-development-update-may-2022.md]: Updated. Please review.

**Pull request merged**